sound/soc/sof/debug.c

```diff
@@ -28,6 +268,26 @@ static ssize_t sof_dfsentry_read(struct file *file, char __user *buffer,
 	int size;
 	u8 *buf;
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST)
+	if (!strcmp(dfse->dfsentry->d_name.name, "ipc_flood_count") ||
+	    !strcmp(dfse->dfsentry->d_name.name, "ipc_flood_duration_ms")) {
+		/* copy to userspace */
+		if (dfse->cache_buf) {
+			if (!*ppos) {
```

Comment:
Change this to
```
if (*ppos)
	return 0;
```
to remove one more indentation level

Reply:
+1